files/image_config/sudoers/sudoers

```diff
@@ -60,6 +60,9 @@ ALL     ALL=NOPASSWD: READ_ONLY_CMDS
 # Prevent password related command into syslog
 Defaults!PASSWD_CMDS !syslog
 
+# Make sure sudo password prompt times out after 5 mins
+Defaults passwd_timeout=5
```

Comment:
>passwd_timeout [](http://example.com/codeflow?start=9&length=14)

Is it a well known issue in upstream? Do you have a tracking link for this bug?

Reply:
sonic bug [https://msazure.visualstudio.com/One/_workitems/edit/13121024/](https://msazure.visualstudio.com/One/_workitems/edit/13121024/)